### ISSUE TITLE
Two way binding

### ISSUE THREAD

**Issue by ImpossivelPT** [pencilpix/vue2-clock-picker#1]
Hi there, 

I've tested your component and really like it, 
but there is one problem that makes it not suitable for my case, 
the problem is that if the value of the time is changed programmatically the input of your picker does not update.

This might be because the value of the prop "value" is being set on data->inputValue and this only happens when the component renders, the solution might be referring to the prop "value" along the component instead of referring to InputValue, making then the component dynamic.

Was this something you consider doing?

Thanks!

**Comment by pencilpix** [comment]
HI @ImpossivelPT 
I will see what can solve this bug ASAP.

**Comment by ImpossivelPT** [comment]
Thanks a lot! There is a need for a good time picker for Vue!

Em 01/02/2018 19:09, "Mohamed Hassan" <notifications@github.com> escreveu:

**Comment by pencilpix** [comment]
@ImpossivelPT I couldn't use prop directly since I have to change the value of the input when selected from dialog. and you know the limitation of Vuejs that prevent reassign value of a prop inside the child component and will throw an error!

So `v0.0.7` I made the component aware of the prop `value` changes but not two-way bindings. using that you are free to update the value outside the component directly or use the method `setValue` to change the time. but the changes that returned from the dialog won't change the value of the prop.

I hope that is helpful. I closed the issue. and feel free to open if there is any issue.
Thanks a lot
